src/config/ConfigService.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "nlohmann/json.hpp"
+#include "configData.h"
+
+using json = nlohmann::json;

```

Comment:
```suggestion
using nlohmann::json;
```
Насколько я понимаю, это одно и то же, но второе понятнее и компактнее